### ISSUE TITLE
Video sharing on Steem

### ISSUE THREAD

**Issue by jsigwart** [lbryio/spee.ch#864]
Trying to embed a spee.ch video with the social sharing iframe results in Unsupported file type instead of embedding the video:

<img width="874" alt="screen shot 2019-01-08 at 6 50 36 pm" src="https://user-images.githubusercontent.com/1363664/50875244-51289f80-137c-11e9-842e-de597ebb737b.png">

**Comment by skhameneh** [comment]
That's interesting, very unexpected!
Thanks for filing

**Comment by kauffj** [comment]
@jsigwart could or did you verify that Steem does let you embed other iframe based videos?

**Comment by tzarebczan** [comment]
Ah, think I found the issue...steemit has to whitelist it: https://github.com/steemit/condenser/issues/271

**Comment by jsigwart** [comment]
@kauffj Yes, I confirmed Steem has a condenser that allows videos from specific sites to be shown, and here is what was done to enable Twitch videos on Steem: https://github.com/steemit/condenser/pull/3088/files

**Comment by tzarebczan** [comment]
@kodxana has a branch going on https://github.com/kodxana/condenser/commits/master which I reviewed recently. I asked him to fix up / PR it.

**Comment by jsigwart** [comment]
@kodxana Tom gave you some feedback and said that you need to do the PR with Steemit so they can give additional feedback - can you please give an update on this one?

**Comment by tzarebczan** [comment]
The pr is here: https://github.com/steemit/condenser/pull/3167

Looks like it's waiting for review. Maybe @kauffj can give someone at steemit a nudge :)

**Comment by kauffj** [comment]
It shows failing tests. Can we fix those issues before nudging?

**Comment by jessopb** [comment]
Where are we on this?

**Comment by alyssaoc** [comment]
@tzarebczan can you update status/next steps
